go-runtime/ftl/ftltest/ftltest.go

```diff
@@ -521,9 +489,39 @@ func CallEmpty[VerbClient any](ctx context.Context) error {
 	return err
 }
 
+// GetDatabaseHandle returns a database handle for the given database name. Database names are transformed
+// to lower camel when provisioned and represented in the schema.
+//
+// E.g.
+//
+//	type MyDatabase = ftl.PostgresDatabase
+//
+// can be referenced as "myDatabase"
+func GetDatabaseHandle(ctx context.Context, dbname string) (ftl.DatabaseHandle, error) {
```

Comment:
this sucks- the goal is to provide a helper function that lets users query their databases in tests. ideally we'd be able to derive the db name from the type so given 
```
type testdb = ftl.PostgresDatabaseHandle
```
 you could do something like 
 ```
 ftltest.GetDatabaseHandle[testdb]()
 ```
instead we're messing with strings, `ftltest.GetDatabaseHandle("testdb")`.
 
 unfortunately because this aliasing pattern does not create a new type, we can't derive the db name from reflection at runtime (the alias name is lost). we have the same problem elsewhere, but only in codegenned files where it doesn't feel as bad to programmatically provide the strings